### DISCUSSION TITLE
Keycloak implementation

### DISCUSSION THREAD

**Discussion by Stoica-Mihai** [cherrypy/cherrypy#1869] [Q&A] [answered]
Hello,

  Is there any documentation on how I could implement keycloak with cherrypy? I've looked everywhere and no such luck.

Best regards.

**Comment by webknjaz** [comment, marked as answer]
Implementing keycloack may mean different things to people with different needs. CherryPy is a low-level framework that does not make such decisions for you letting you have all the power over how you want things to be implemented. You'll likely to have to figure out what works for you best. You may also try chatting with folks on Gitter if you have concrete examples of what you're attempting to do.

**Reply by Stoica-Mihai** [reply]
Thanks for pointing out Gitter. I will try my luck there. 

Best regards.
